src/components/Gauge/Gauge.tsx

```diff
@@ -60,7 +63,7 @@ export const Gauge: React.FC<Props> = (props) => {
         <div {...rest} {...block(props)}>
             <svg {...elem('svg', props)} width="100%">
                 <defs>
-                    <linearGradient id={`Gauge__gradient--${context}`}>
+                    <linearGradient id={`Gauge__gradient--${context}-${id.current}`}>
```

Comment:
I don't think `${context}` is relevant any longer since every definition is now unique

Reply:
...apart from that, LGTM.

Reply:
it doesn't harm

Reply:
I don't understand. Why keep code that doesn't serve a purpose?
Besides, it _does_ "harm": it's misleading.

Reply:
I'm not following what do you mean. It is still the colors of that context, so the name is accurate. It is only the name, where the ID is added, is unique. 
To say the same thing in an other way, `${context}` is as redundant as `Gauge` and `gradient`.

Reply:
There is only a single gradient definition per instance of `Gauge`. This could make it seem as if there would be one per context. It serves no purpose. "Gauge" and "gradient" are just names - not variables.
It's not really productive to continue this debate at length so if you can't follow then go ahead.